plugins/woocommerce/client/admin/client/task-lists/fills/Marketing/Plugin.tsx

```diff
@@ -26,8 +27,30 @@ export type PluginProps = {
 	manageUrl?: string;
 	name: string;
 	slug: string;
+	tags?: TagsSlug[];
+	learnMoreLink?: string;
+	installExternal?: boolean;
 };
 
+const tagsToPillsMap = {
+	marketplace: __( 'Marketplace', 'woocommerce' ),
```

Comment:
I'm unsure if we should have this in API. My thinking is that pills need to be pre-defined, since otherwise it might be prone to errors like unintentional multiple pills displayed, breaking the UI

Reply:
Defined the map here looks good to me!